From da9cd7d482206ae8bcacc1e1ac7c4e00d8c49a68 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Sep 2016 13:39:44 +0500
Subject: [PATCH] obey CMAKE_INSTALL_PREFIX

Unsetting CMAKE_INSTALL_PREFIX breaks relative installation
paths. The recent updates to cmake.bbclass requires
using the installation prefix to allow installing to
correct hierarchy under yocto environment.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 898b0f9..1451512 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,8 +57,6 @@ if(NOT WIN32)
     include(FindPkgConfig)
 endif()
 
-set (CMAKE_INSTALL_PREFIX "")
-
 if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_C_COMPILER_ID MATCHES "Clang")
     set(COMMON_COMPILE_FLAGS "-Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers")
     set(COMMON_COMPILE_FLAGS "${COMMON_COMPILE_FLAGS} -fno-strict-aliasing -fno-builtin-memcmp")
-- 
1.9.1

